src/test/java/com/lafaspot/icap/client/IcapClientTest.java

```diff
@@ -27,6 +27,7 @@
 import org.testng.annotations.BeforeClass;
```

Comment:
rename this class to IcapClientIT
create a new class called IcapClientTest and add your test there